config/locales/en.yml

```diff
@@ -17,3 +17,9 @@ en:
     permission_priority: "Priority"
     permission_visible: "Visible"
     permission_boolean: "Boolean"
```

Comment:
I mean lets add hint text what boolean, visible, priority mean here and what would be the impact.